### ISSUE TITLE
Add in a link to subscribe w/o commenting (via Postmatic)

### ISSUE THREAD

**Issue by Vernal** [postmatic/epoch#60]
We used to have this built into the PM widget. Maybe we can resurrect it easily enough.

**Comment by solhuebner** [comment]
:+1:

**Comment by Vernal** [comment]
Hey @cyberhobo we're booting up Epoch dev again. What do you think of this?

**Comment by cyberhobo** [comment]
Why put it in epoch instead of Postmatic?

**Comment by Vernal** [comment]
Because I recollected that we talked about it once and you said it is
something that should go into Epoch. But maybe I dreamt that. Whatchoo
think?

On September 9, 2016 at 1:19:06 PM, Dylan Kuhn (notifications@github.com)
wrote:

Why put it in epoch instead of Postmatic?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/postmatic/epoch/issues/60#issuecomment-245979165, or mute
the thread
